### PR TITLE
Log deprecated metrics

### PR DESCRIPTION
**Pull request by yngvar-antonsson**

### CODE REVIEW

File: metrics/init.lua [hunk 1]
```diff
@@ -119,6 +119,9 @@ return {
     invoke_callbacks = invoke_callbacks,
     set_global_labels = set_global_labels,
     enable_default_metrics = function(include, exclude)
+        log.warn('metrics tnt_net_sent_rps, tnt_net_received_rps, tnt_net_connections_rps, '..
+            'tnt_net_requests_rps, tnt_stats_op_rps ' ..
+            'is deprecated and will be removed in next releases.')
```
Comment (DifferentialOrange) [inline metrics/init.lua:124]:
```suggestion
            'are deprecated and will be removed in next releases.')
```

**Comment by DifferentialOrange** [comment]
Follows up https://github.com/tarantool/metrics/pull/339

**Comment by Totktonada** [comment]
Where I can read details and reasoning behind?

**Comment by DifferentialOrange** [comment]
> Where I can read details and reasoning behind?

https://github.com/tarantool/metrics/pull/339

**Pull request merged**